+adi/+internal/StingrayControl.m

```diff
@@ -18,69 +18,50 @@
     end
     
     properties(Hidden)
-        SRayCtrlDeviceNames = {'one-bit-adc-dac'};
-        SRayCtrlDevLabel = 'stingray_control';
-        SRayCtrlDevices
+        SRayCtrlDeviceName = 'stingray_control';
+        SRayCtrlDevice
     end
         
     methods
         function result = get.PowerUpDown(obj)
             result = false;
-            if ~isempty(obj.SRayCtrlDevices)
-                result = obj.getAttributeRAW('voltage5', 'raw', true, obj.SRayCtrlDevices);
+            if ~isempty(obj.SRayCtrlDevice)
+                result = obj.getAttributeRAW('voltage5', 'raw', true, obj.SRayCtrlDevice);
             end
         end
         
         function set.PowerUpDown(obj, value)
-            obj.setAttributeRAW('voltage5', 'raw', num2str(value), true, obj.SRayCtrlDevices);
+            obj.setAttributeRAW('voltage5', 'raw', num2str(value), true, obj.SRayCtrlDevice);
```

Comment:
Shouldn't this be protected by a ConnectedToHardware check?

Reply:
Fixed.